xmtp_mls/src/groups/group_metadata.rs

```diff
@@ -150,8 +150,12 @@ impl TryFrom<DmMembersProto> for DmMembers {
 
     fn try_from(value: DmMembersProto) -> Result<Self, Self::Error> {
         Ok(Self {
-            member_one_inbox_id: value.dm_member_one.unwrap().inbox_id.clone(),
-            member_two_inbox_id: value.dm_member_two.unwrap().inbox_id.clone(),
```

Comment:
These unwraps were just on the mls_dms feature branch already. Just cleaning up the linter in this PR. cc @cameronvoell